cg/meta/workflow/balsamic.py

```diff
@@ -198,6 +199,24 @@ def get_verified_bed(self, sample_data: dict, panel_bed: Path) -> Optional[str]:
                 )
             return Path(self.bed_path, target_bed).as_posix()
 
+    @staticmethod
+    def get_verified_pon(panel_bed: Path, pon_cnn: str) -> Optional[str]:
+        """Returns the validated PoN path
+
+        Raises BalsamicStartError:
+            When there is a missmatch between the PoN and the panel bed file names
+        """
+        if pon_cnn:
+            pon_cnn = Path(str(pon_cnn))
+            if os.path.basename(panel_bed).split(".")[0] not in os.path.basename(pon_cnn):
```

Comment:
Was this hard to do with Pathlib or why the extra os.path import?

Reply:
On the contrary, it's much simpler and readable: [d63a73d](https://github.com/Clinical-Genomics/cg/pull/1477/commits/d63a73d705beb80d22a86fdb9644351d77003e97)